src/themes/blueberry/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+
+import { theme as defaultTheme } from '../../layout'
+import ExampleTypeConstructor from '../../exampleTypes/'
+
+import Input from './Input'
+import Checkbox from './Checkbox'
+import Fonts from './Fonts'
+import Style from './Style'
+import Table from './Table'
+import Button from './Button'
+import ButtonRadio from './ButtonRadio'
+import ListGroupItem from './ListGroupItem'
+import Wrap from './Wrap'
+import TagComponent from './TagComponent'
+import PagerItem from './PagerItem'
+import Adder from './Adder'
+
+export let Pager = props => (
+  <ExampleTypes.ResultPager Item={PagerItem} {...props} />
+)
+
+let baseTheme = {
+  ...defaultTheme,
+  Adder,
+  'Adder.Item': ListGroupItem,
+  Wrap,
+  Fonts,
+  Style,
+  Table,
+  Button,
+  Input,
+  Checkbox,
+  RadioList: ButtonRadio,
+  'FieldPicker.Item': ListGroupItem,
```

Comment:
This is evidence of a bug where nested themes don't work when the name a component is given by `withTheme` (or `ThemeConsumer`) is different from its key in the theme.

If the line in `src/layout/index.js` is uncommented (`FieldPicker` is set to `NestedPicker` in the default layout theme), this override won't be merged, because `withTheme('NestedPicker')` does not recognize keys starting with `'FieldPicker'`.

Reply:
Not sure I follow here - this is by design 🤔 Things that can overriden in a theme should have a single, canonical name.